src/cmd/cmd.zig

```diff
@@ -735,7 +735,13 @@ fn validator() !void {
         allocator.destroy(gossip_service);
     }
 
-    const geyser_writer = try buildGeyserWriter(allocator, app_base.logger.unscoped());
+    const geyser_writer: ?*GeyserWriter = createGeyserWriterFromConfig(
+        allocator,
+        config.current.geyser,
+    ) catch |err| switch (err) {
+        error.GeyserWriterIsDisabled => null,
```

Comment:
Nit: Previously implementation logs `GeyserWriter is disabled.` when geyser is disabled.